SpatialGDK/Source/SpatialGDK/Public/Interop/GlobalStateManager.h

```diff
@@ -114,11 +119,8 @@ class SPATIALGDK_API UGlobalStateManager : public UObject
 	UPROPERTY()
 	USpatialStaticComponentView* StaticComponentView;
 
-	UPROPERTY()
-	USpatialSender* Sender;
-
-	UPROPERTY()
-	USpatialReceiver* Receiver;
+	TUniquePtr<SpatialGDK::ClaimPartitionHandler> ClaimHandler;
```

Comment:
Why is the partition handler memory managed differently than the query handler?

Reply:
`ClaimPartition` doesn't have to be a UniquePtr, but it does need to be initialized after construction as it depends on an instance of `Connection`. I considered using a `TOptional` here but opted for `TUniquePtr` as it's used a lot in the NetDriver.